infrastructure/patent_watch.tf

```diff
@@ -0,0 +1,45 @@
+resource "aws_lambda_function" "patent_notifier" {
+  function_name = "patent_notifier_s3"
+  filename       = "x.zip"
+  role          = "arn:aws:iam::467749311079:role/LabRole"
+  handler       = "patent-s3.handler"
+  timeout       = 300
+  runtime       = "python3.9"
+  layers        = [aws_lambda_layer_version.requests_layer.arn]
+  source_code_hash = filebase64sha256("patent_notifier.zip")
```

Comment:
the same as in the last pr 

filename and hash are different and there is no code

you can also use your role from locals :)